src/pages/registration/contributors_tab/components/EditAffiliationModal.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Typography } from '@mui/material';
+import { useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useFetchOrganization } from '../../../../api/hooks/useFetchOrganization';
+import { SelectInstitutionForm } from '../../../../components/institution/SelectInstitutionForm';
+import { SelectInstitutionSkeleton } from '../../../../components/institution/SelectInstitutionSkeleton';
+import { Modal } from '../../../../components/Modal';
+import { setNotification } from '../../../../redux/notificationSlice';
+import { Affiliation } from '../../../../types/contributor.types';
+import { Organization } from '../../../../types/organization.types';
+import { SpecificContributorFieldNames } from '../../../../types/publicationFieldNames';
+import { Registration } from '../../../../types/registration.types';
+import { findDescendantWithId, getTopLevelOrganization } from '../../../../utils/institutions-helpers';
+
+interface EditAffiliationModalProps {
+  affiliationModalIsOpen: boolean;
+  toggleAffiliationModal: () => void;
+  affiliationToEdit: Organization;
+  baseFieldName: string; // TODO: This can be imported instead
```

Comment:
utdatert kommentar? antar vi må sende denne som prop da den er `contributors[index]`?
```suggestion
  baseFieldName: string;
```

Reply:
Ja, jeg skrev den til meg selv, men oppdaget etterpå at det ikke gikk